package.json

```diff
@@ -103,6 +103,7 @@
     "source-map-support": "^0.5.21",
     "steam-shortcut-editor": "^3.1.3",
     "tslib": "^2.8.1",
+    "unzipper": "^0.12.3",
```

Comment:
do we need this dependency? I know we unarchive files in other places like when we download wine/proton, can't we unzip without an extra dependency?